playbooks/roles/forum/templates/forum-watchdog.sh.j2

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+{% if ! devstack %}
+  tail -f {{ supervisor_log_dir }}/forum-stderr.log |
+  grep --line-buffered 'Mongo::Error::OperationFailure - not master and slaveOk=false (13435)' |
+  while read;
```

Comment:
Just a small nit: isn't the semicolon redundant here if aren't planning to put `do` in the same line as `while`?

Reply:
Just fixed this